terraria/vendor_images

```diff
@@ -0,0 +1 @@
+ryshe/terraria:latest
```

Comment:
Probably shouldn't use latest here since then we can never push out an update to users.